openmoc/numpy_typemaps.i

```diff
@@ -75,3 +75,9 @@
 
 /* The typemap used to match the method signature for Solver::setFluxes */
 %apply (FP_PRECISION* INPLACE_ARRAY1, int DIM1) {(FP_PRECISION* in_fluxes, int num_fluxes)}
+
+/* The typemap used to match the method signature for
+ * Geometry::getSpatialDataOnGrid */
+%apply (double* IN_ARRAY1, int DIM1) {(double* gridx, int numx)}
```

Comment:
Can `numx` be made `num_x` (and same for numy and numxy) here and in `Geometry.cpp` to be consistent with how other methods have `num_x` as inputs?